cypress/support/e2e.js

```diff
@@ -38,6 +38,7 @@ const KNOWN_ERRORS = [
   // This link then navigates the test to the BBC News Pidgin WhatsApp channel
   // https://www.whatsapp.com/channel/0029VawW7AjFy726zWLIFq2j
   'requireLazy is not defined',
+  'ResizeObserver loop completed with undelivered notifications',
```

Comment:
This is the error message which it being generated from within https://www.bbc.com/uzbek/articles/c4g8ex7ked1o/cyr